docs/main-concepts/whats-a-fetch.md

```diff
@@ -0,0 +1,191 @@
+---
+id: whats-a-fetch
+title: What's a fetch?
+sidebar_label: What's a fetch?
+---
+
+> **A small disclaimer:** Naming collisions are an unfortunate thing we have to deal with in software engineering and ReSift's "fetches" are no different. For now, please forget what you know about fetches or [`window.fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) because they are _different_ from the fetches we define here.
+
+## A fetch is like an order
+
+ReSift has this concept of **fetches** that are loosely analogous to **orders**.
+
+Generally speaking, an **order** is a noun we use to talk about the process of many things revolving around placing and receiving an order including:
```

Comment:
Generally speaking, an **order** is a noun we use to describe multiple pieces of information resulting from walking through an ordering process, including: